projboard/tests/test_client.py

```diff
@@ -111,3 +111,27 @@ def test_aboutpage(client):
     template_names = set(tmpl.origin.template_name for tmpl in response.templates)
     # And check if that 'about/about.html' is in the set.
     assert 'about/about.html' in template_names
+
+
+@pytest.mark.django_db
+def test_get_readArticlePage(client, article):
```

Comment:
If you create the article, 
and search return home page with article, please assert the fixture content and the result returned by view

Reply:
done